src/web/views/select-address-form.njk

```diff
@@ -27,6 +27,13 @@
         {% if addresses | length === 0 %}
             <h2 class="govuk-heading-m">{{ addressNotFound }}</h2>
         {% else %}
+
+            {% set isSelected = false if addressSelected else true %}
+
+            {% set firstOption = { text: addresses | length + " addresses found", value: "", disabled: true, selected: isSelected } %}
```

Comment:
To translate this I think it will need to be constructed in `behaviourForGet` and added to `res.locals` using `req.t()`

Reply:
thanks - done